### PR TITLE
#28 Fix bug 

### PR DESCRIPTION
**Pull request by Evgentik**
Update models, db, jscripts

### CODE REVIEW

File: Recrutify/Recrutify.Host/Configuration/Profiles/ProjectProfile.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public ProjectProfile()
             CreateMap<Project, ProjectDTO>().ReverseMap();
             CreateMap<StaffDTO, Staff>().ReverseMap();
             CreateMap<PrimarySkill, PrimarySkillDTO>().ReverseMap();
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Configuration/Profiles/ProjectProfile.cs:16]: Remove reversemap here

File: Recrutify/Recrutify.Host/libman.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1.0",
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/libman.json:2]: Think its extra file, remove it

Comment (Evgentik) [reply Recrutify/Recrutify.Host/libman.json:2]: ok. I don't know how he appeared

File: Recrutify/Recrutify.Host/Controllers/OData/ProjectsController.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OData.Query;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using Recrutify.Services.DTOs;
+using Recrutify.Services.Services.Abstract;
+
+namespace Recrutify.Host.Controllers.OData
+{
+    public class ProjectsController : ODataController
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/OData/ProjectsController.cs:11]: Remove Odata from this PR

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**